doc/articles/getting-started/requirements.md

```diff
@@ -34,4 +34,4 @@ Uno Platform applications run on all macOS versions supported by Mac Catalyst -
 
 ## Linux
 
-Uno Platform applications run on Linux distributions and versions where .NET 7 is supported, [listed here](https://docs.microsoft.com/en-ca/dotnet/core/install/linux).
\ No newline at end of file
+Uno Platform applications run on Linux distributions and versions where .NET 7 is supported, [listed here](https://docs.microsoft.com/en-ca/dotnet/core/install/linux).
```

Comment:
```suggestion
Uno Platform applications run on Linux distributions and versions where .NET 7 is supported, [listed here](https://learn.microsoft.com/dotnet/core/install/linux).
```

Reply:
@morning4coffe-dev  - this should be ".NET 7 or later. "